src/SvelteTable.svelte

```diff
@@ -109,10 +109,10 @@
   if (!Array.isArray(expanded)) throw "'expanded' needs to be an array";
   if (!Array.isArray(selected)) throw "'selection' needs to be an array";
   if (expandRowKey !== null) {
-    console.warn("'expandRowKey' is depricated in favour of 'rowKey'");
+    console.warn("'expandRowKey' is deprecated in favour of 'rowKey'");
   }
   if (classNameRowSelected && !rowKey)
-    console.warn("'rowKey' is needed to use 'classNameRowSelected'");
+    console.error("'rowKey' is needed to use 'classNameRowSelected'");
```

Comment:
I'd like to keep it as `warn` until functionality is removed and not useable

Reply:
Ready for second code review